src/azure-cli-core/azure/cli/core/_profile.py

```diff
@@ -12,8 +12,10 @@
 import os.path
 import re
 import string
+import traceback
 from copy import deepcopy
 from enum import Enum
+from requests.exceptions import InvalidURL
```

Comment:
Always use lazy import to avoid performance impact. See #13843

Reply:
done